packages/core/src/components/icon-button/icon-button.tsx

```diff
@@ -21,6 +22,12 @@ export type IconButtonVariant = ButtonVariant;
 export class IconButton implements Button {
   @Element() hostElement: HTMLIxIconButtonElement;
 
+  /**
+   * Accessibility label for the icon button
+   * @since 1.7.0
+   */
+  @Prop() ixAriaLabel: string;
```

Comment:
Would it be possible to rename the porperty to something like `a11yLabel` or just `ariaLabel`?

Reply:
Renaming the property "ixAriaLabel" to "a11yLabel" is unproblematic. Corresponding properties of other components should also be renamed accordingly to ensure consistency. As there is already a standard aria attribute called "aria-label", the use of a property label "ariaLabel" could possibly lead to conflicts, as I understand that there are automatic mechanisms for converting camel case notations into kebab case notations or vice versa.